---
layout: default
title: Selected Papers
---

<br>
<br>

<p style="padding:3px; color: black; background-color: #FEF9E7; border: #F2F3F4 1px solid">List of publications available on my <a href="https://www.webofscience.com/wos/author/record/1592919">Web of Science profile.</a>
</p>

<br>

<div id="toptitle">
	<h1>{{ page.title }}</h1>
</div>


<a href="https://aacrjournals.org/bloodcancerdiscov/article/4/4/318/727506/Patient-Derived-iPSCs-Faithfully-Represent-the">Patient-Derived iPSCs Faithfully Represent the Genetic Diversity and Cellular Architecture of Human Acute Myeloid Leukemia.</a>
<i> Blood cancer discovery</i>, 4(4), 318–335, 2023.
<br>
A.G. Kotini, S. Carcamo, N. Cruz-Rodriguez, ... <u>E. Bernard</u> ... , D. Hasson, E.P. Papapetrou.
<br>
<br>
<br>



<a href="https://www.nature.com/articles/s41467-022-32103-8">Unified classification and risk-stratification in Acute Myeloid Leukemia.</a>
<i> Nature Communications</i>, 13, 4622, 2022.
<br>
Y. Tazi, J. E. Arango-Ossa, Y. Zhou, <u>E. Bernard</u>, et al.
<br>
<br>
<br>


<a href="https://evidence.nejm.org/stoken/default+/JEUNBGEPS5SXRMWCYYTJ/full?redirectUri=/doi/full/10.1056/EVIDoa2200008">Molecular International Prognostic Scoring System for Myelodysplastic Syndromes.</a>
<i> NEJM Evidence</i>, 1 (7), 2022.
<br>
<u>E. Bernard</u>, H. Tuechler, P. G. Greenberg, et al.
<br>
<a href="https://github.com/papaemmelab/ipssm"><small style="color:red">R Package.</small></a>
<a href="https://mds-risk-model.com/"><small style="color:red">Web Calculator.</small></a>
<a href="https://github.com/papaemmelab/IPSSMstudy"><small style="color:red">Reproduce.</small></a>
<br>
<br>
<br>


<a href="https://ashpublications.org/bloodadvances/article/doi/10.1182/bloodadvances.2021006325/483554/Patient-specific-MDS-RS-iPSCs-define-the-mis">Patient-specific MDS-RS iPSCs define the mis-spliced transcript repertoire and chromatin landscape of SF3B1-mutant HSPCs.</a>
<i>Blood Advances</i>, 6 (10): 2992–3005, 2022.
<br>
G. Asimomitis, A. G. Deslauriers, A. G Kotini, <u>E. Bernard</u>, et al.
<br>
<br>
<br>


<a href="https://bloodcancerdiscov.aacrjournals.org/content/2/5/500">ZBTB33 is mutated in clonal hematopoiesis and myelodysplastic syndromes and impacts RNA Splicing.</a>
<i> Blood Cancer Discovery</i>, 2(5), 500–517, 2021.
<br>
E.M. Beauchamp, M. Leventhal, <u>E. Bernard</u>, et al.
<br>
<br>
<br>


<a href="https://www.nature.com/articles/s41467-020-20565-7">Interplay between chromosomal alterations and gene mutations shapes the evolutionary trajectory of clonal hematopoiesis.</a>
<i> Nature Communications</i>, 12, 338, 2021.
<br>
T. Gao, R. Ptashkin, K. L. Bolton, ... , <u>E. Bernard</u>, ... , S. M. Devlin, E. Papaemmanuil.
<br>
<br>
<br>


<a href="https://www.nature.com/articles/s41588-020-00710-0">Cancer therapy shapes the fitness landscape of clonal hematopoiesis.</a>
<i> Nature Genetics</i>, 52, 1219–1226, 2020.
<br>
K. L. Bolton, R. N. Ptashkin, T. Gao, ... , <u>E. Bernard</u>, ... , A. Zehir, E. Papaemmanuil.
<br>
<br>
<br>


<a href="https://www.nature.com/articles/s41591-020-1008-z">Implications of TP53 Allelic State for Genome Stability, Clinical Presentation and Outcomes in Myelodysplastic Syndromes.</a>
<i> Nature Medicine</i>, 26, 1549–1556, 2020.
<br>
<u>E. Bernard</u>, Y. Nannya, R. P. Hasserjian, et al.
<br>
<a href="https://www.biorxiv.org/content/10.1101/2019.12.19.868844v1"><small style="color:red">Preprint.</small></a>
<a href="https://github.com/papaemmelab/MDS-TP53-state"><small style="color:red">Reproduce.</small></a>
<br>
<br>
<br>


<a href="https://onlinelibrary.wiley.com/doi/10.1002/ajh.25641">Stability and uniqueness of clonal immunoglobulin CDR3 sequences for MRD tracking in multiple myeloma.</a>
<i> Am. J. Hematol.</i>, doi: 10.1002/ajh.25641, 2019.
<br>
E. H. Rustad, K.Misund, <u>E. Bernard</u>, et al.
<br>
<br>
<br>


<a href="https://doi.org/10.1080/10428194.2019.1627542">Multifaceted modes of action of azacytidine: a riddle wrapped up in an enigma.</a>
<i>Leukemia & Lymphoma,</i> 12:1-5, 2019.
<br>
E. Lamprianidou, E. Zoulia, <u>E. Bernard</u>, C. Kordella, M. Papoutselis, Z. Bezirgiannidou, G. Vrachiolias, E. Papaemmanuil and I. Kotsianidis.
<br>
<br>
<br>


<a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2004045">A molecular atlas of the developing ectoderm defines neural, neural crest, placode and non-neural progenitor identity in vertebrates.</a>
<i>PLoS Biology,</i> 15(10): e2004045, 2017.
<br>
J.-L. Plouhinec, S. Medina-Ruiz, C. Borday, <u>E. Bernard</u>, J.-P. Vert, M. B. Eisen, R. M. Harland and A. H. Monsoro-Burq.
<br>
<br>
<br>


<a href="http://onlinelibrary.wiley.com/doi/10.1002/minf.201700053/abstract">Kernel multitask regression for toxicogenetics.</a>
<i>Molecular Informatics,</i> 36(10), 2017.
<br>
<u>E. Bernard</u>, Y. Jiao, E. Scornet, V. Stoven, T. Walter and J.-P. Vert. 
<br>
   <a href="http://www.biorxiv.org/content/early/2017/08/01/171298"><small style="color:red">Preprint.</small></a>
   <a href="https://github.com/jpvert/kmr"><small style="color:red">R Package.</small></a>
   <a href="https://github.com/jpvert/kmr4toxicogenetics"><small style="color:red">Reproduce.</small></a>
<br>
<br>
<br>


<a href="http://www.biomedcentral.com/1471-2105/16/262">A convex formulation for joint RNA isoform detection and quantification from multiple RNA-seq samples.</a>
<i>BMC Bioinformatics,</i> i16:262, 2015.
<br>
<u>E. Bernard</u>, L. Jacob, J. Mairal, E. Viara and J.-P. Vert.
<br>
   <a href="https://hal-mines-paristech.archives-ouvertes.fr/hal-01123141"><small style="color:red">Preprint.</small></a>
<br>
<br>
<br>


<a href="http://bioinformatics.oxfordjournals.org/content/early/2014/06/26/bioinformatics.btu317">Efficient RNA isoform identification and quantification from RNA-seq data with network flows.</a>
<i>Bioinformatics, </i>30(17):2447-2455, 2014.
<br>
<u>E. Bernard</u>, L. Jacob, J. Mairal and J.-P. Vert.
<br>
<a href="http://hal-ensmp.archives-ouvertes.fr/hal-00803134"><small style="color:red">Preprint.</small></a>
<br>
<br>
<br>


<a href="http://journals.ametsoc.org/doi/abs/10.1175/JCLI-D-12-00836.1">Clustering of maxima: spatial dependencies among heavy rainfall in france.</a>
<i> Journal of Climate,</i> 26, 7929-7937, 2013.
<br>
<u>E. Bernard</u>, P. Naveau, M. Vrac and O. Mestre.
<br>
<br>
<br>


<a href="http://www.cims.nyu.edu/~shafer/papers/SmithBernard13.pdf">Geostrophic turbulence near rapid changes in stratification.</a>
<i>Physics of Fluids,</i> 25, 046601, 2013.
<br>
K.S. Smith and <u>E. Bernard</u>.
<br>
<br>


<h1>Thesis</h1>
</div>

E. Bernard. 
<a href="https://hal.archives-ouvertes.fr/tel-01681314/">Deciphering splicing with sparse regression techniques in the era of high-throughput RNA sequencing.</a>
<i>Phd thesis,</i> 2016.
<br>
<br>
